### DISCUSSION TITLE
New Update Feature doesn't allow you to set scores on DMCA'ed maps

### DISCUSSION THREAD

**Discussion by zesteas** [ppy/osu#19527] [Q&A] [answered]
Some DMCA'ed maps will not allow you to update the beatmap, while those with the mp3 removed entirely will let you submit scores only if you remove the mp3. Could there be a way to change how updating works for certain maps to allow score submission?

https://user-images.githubusercontent.com/110361637/182181961-96843d1a-4b2f-426e-b0fd-b31864a756cf.mp4

**Comment by peppy** [comment, marked as answer]
The data is not available on the server, so updating is not possible. Please play another beatmap.
